File: Lista_01_POO/Cinema/QuestaoCinema.java
import java.util.Scanner;

class Cinema{
    String dia;
    int horario;

    public double ingressoDia(){
        if (dia.equals("Segunda") || dia.equals("Terça")|| dia.equals("Quinta") && horario > 17){
            if (horario >= 17){
                return 24;
            } else {
                return 16;
            }
        } else if (dia.equals("Sexta") || dia.equals("Sábado") || dia.equals("Domingo")) {
            if (horario >= 17) {
                return 30;
            } else {
                return 20;
            }
        } else{
            return 8;
        }
    }

}

public class QuestaoCinema{
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        Cinema dadosCinema = new Cinema();

        System.out.print("Digite o dia da sessão(ex: Domingo): ");
        dadosCinema.dia = teclado.next();

        System.out.print("Digite o horário da sessão(ex: 15:30): ");
        String horarioString = teclado.next();
        String[] horarios = horarioString.split(":");
        dadosCinema.horario = Integer.parseInt(horarios[0]);

        System.out.println(String.format("Valor do ingresso = R$ %.2f", dadosCinema.ingressoDia()));
    }
}

File: POOLists/Lista01/Cinema/QuestaoCinema.py
class Cinema:
    def __init__(self):
        self.dia = ""
        self.horario = 0

    def ingressoDia(self):
        if (self.dia == "Segunda" or self.dia == "Terça" or self.dia == "Quinta"):
            if (self.horario >=17):
                return 24
            else:
                return 16
        elif (self.dia == "Sexta" or self.dia == "Sábado" or self.dia == "Domingo"):
            if (self.horario >= 17):
                return 30
            else:
                return 20;
        else:
            return 8
        

c = Cinema()
c.dia = input("Digite o dia da sessão (ex: Domingo): ")
horarioString = input("Digite o horário da sessão (ex: 15:30): ")
horario = list(horarioString.split(":"))
c.horario = int(horario[0])

print (f"Valor do ingresso = R$ {c.ingressoDia():.2f}")

